chapters/fa/glossary/1.mdx

```diff
@@ -110,20 +114,21 @@
 | Project, as in math | پروجکت؟ |
 | Embedding | embedding? |
 | Tokenized | توکِن‌شده |
+| Mask Filling | پر کردن جاهای خالی متن |
 | Attention Mechanism | مکانیزم توجه |
 | Classification | دسته‌بندی |
 | Attribute, as for a class in code | ویژگی؟ |
 | Label, as in classification | عنوان دسته |
 | Prediction, as in nn model | پیش‌بینی |
+| Bias | سوگیری |
 | Logit, as in math and also in Pytorch | لوجیت؟ |
 | SoftMax | سافت‌مکس |
 | Loss Function | تابع هزینه |
 | Activation Layer | لایه فعال‌سازی |
 | Cross Entropy | آنتروپی متقابل |
 | Head, as in model | سَر |
 | Weight, as in model | وزن |
-| Weights, as in model | وزون؟ |
```

Comment:
Please include the equivalent you suggested for the plural form to avoid confusion in future for other collaborators.